### PR TITLE
Rerun from the start

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipe/migrations/r400.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ __migrator:
   # Only change the migration_number if the bot messes up,
   # changing this will cause a complete rerun of the migration
   migration_number:
-    1
+    2
```
Comment (isuruf) [inline recipe/migrations/r400.yaml:13]: Why?

Comment (CJ-Wright) [reply recipe/migrations/r400.yaml:13]: The bot has messed up a bunch due to various bugs, we aren't far into the graph (maybe 7 PRs) so this isn't too much re-work.

One of the major issues was that to pause the bot we moved the yaml, but this caused rerenders to remove the yaml outright with no way to recover it without a rerun.

Comment (isuruf) [reply recipe/migrations/r400.yaml:13]: Makes sense. Thanks for the info.

Comment (CJ-Wright) [reply recipe/migrations/r400.yaml:13]: The above issue needs a more permanent fix but I'll write that up in an issue.

Comment (CJ-Wright) [reply recipe/migrations/r400.yaml:13]: On the bright side if a PR went in properly then the bot will just move on.

**Pull request merged**